composer/trainer/trainer.py

```diff
@@ -666,6 +681,9 @@ def __init__(
         save_weights_only: bool = False,
         save_num_checkpoints_to_keep: int = -1,
 
+        # Graceful Resumption
+        autoresume: Optional[bool] = False,
```

Comment:
nit: `Optional` means that it can be None, not that it is not required

```suggestion
        autoresume: bool = False,
```